packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -357,6 +367,38 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
                 });
             }
         });
+        registry.registerCommand(OpenEditorsCommands.CLOSE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.editorWidgets.forEach(editor => editor.close())),
+            isEnabled: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length),
+            isVisible: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length)
+        });
+        registry.registerCommand(OpenEditorsCommands.SAVE_ALL_TABS_FROM_TOOLBAR, {
+            execute: widget => this.withOpenEditorsWidget(widget, () => this.commandService.executeCommand(CommonCommands.SAVE_ALL.id)),
+            isEnabled: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length),
+            isVisible: widget => this.withOpenEditorsWidget(widget, () => !!this.editorWidgets.length)
+        });
+        registry.registerCommand(OpenEditorsCommands.CLOSE_ALL_IN_GROUP, {
+            execute: async (id): Promise<void> => {
+                const openEditorsWidget = await this.widgetManager.getOrCreateWidget<OpenEditorsWidget>(OpenEditorsWidget.ID);
+                const widgets = openEditorsWidget.getEditorWidgetsByGroup(id);
+                widgets?.forEach(widget => widget.close());
+            }
+        });
+        registry.registerCommand(OpenEditorsCommands.SAVE_ALL_IN_GROUP, {
+            execute: async (id): Promise<void> => {
+                const openEditorsWidget = await this.widgetManager.getOrCreateWidget<OpenEditorsWidget>(OpenEditorsWidget.ID);
+                const widgets = openEditorsWidget.getEditorWidgetsByGroup(id);
+                widgets?.forEach(widget => {
+                    const saveable = Saveable.get(widget);
+                    saveable?.save();
+                });
+            }
+        });
```

Comment:
I've set the visibility of these commands to false. It seems like they are visible in VSCode, but it doesn't seem like it makes much sense since there's no reference to the Group from the command palette.